inference_video.py

```diff
@@ -14,22 +14,19 @@
 def transferAudio(sourceVideo, targetVideo):    
     import shutil
     import moviepy.editor
-    tempAudioFileName = "./temp/audio.mp3"
+    tempAudioFileName = "./temp/audio.mkv"
```

Comment:
Swith to mkv from mp3 as mkv can handle more codecs. If however a codec that mkv doesn't support it used, the transferAudio function may fail.